src/pages/viewer-app/viewer-app.tsx

```diff
@@ -206,12 +210,12 @@ export const ViewerApp = () => {
     }
 
     const params = parseTilesetUrlParams(mainTileset.url, mainTileset);
-    const { tilesetUrl, token, name, metadataUrl } = params;
+    const { tilesetUrl, token, metadataUrl } = params;
 
     fetchMetadata(metadataUrl);
     fetchFlattenedSublayers(tilesetUrl);
 
-    setName(name);
+    //setName(name);
```

Comment:
If we don't need it just remove. If it is necessary to keep it please add comment.